newIDE/app/src/VariablesList/index.js

```diff
@@ -334,9 +348,12 @@ export default class VariablesList extends React.Component<Props, State> {
           useDragHandle
           lockToContainerEdges
         >
+          {!objVariablesTree.length && this._renderEmpty()}
+          {!!objVariablesTree.length && objVariablesTree}
+
           {!containerVariablesTree.length && this._renderEmpty()}
```

Comment:
And if we keep this, we will render an empty message if instance variables are empty but even if object variables are not empty - which also sounds like something we don't want.